src/api_core/upload_cache.py

```diff
@@ -63,8 +63,9 @@ def addFile(self, fdata, fname):
 
         if byte_cnt > self.maxsize:
             fpath.unlink()
+            max_mb = self.maxsize / (1024 * 1024)
             raise Exception(
-                'Uploaded file size exceeded maximum file size.'
+                'Uploaded file size exceeded maximum file size of {} MB. Please submit a GCDL github issue to request a larger file size limit, or decrease your file size.'.format(max_mb)
```

Comment:
Provide a link to the issue page on the GeoCDL repo? 
'GCDL' to 'GeoCDL'
And please keep lines <80 characters.